app/e2e/packetPage.local.ts

```diff
@@ -0,0 +1,100 @@
+import { test, expect, Locator } from "@playwright/test";
+import { getContentLocator, getPacketPageAccordionSection, selectPacketPageTab } from "./utils";
+
+// Test which are only run against localhost, where we can assume we have the demo dataset packets
+test.describe("Local packet page", () => {
+  const parametersPacketId = "20240829-185440-781be0f3";
+  const dependsPacketId = "20250120-073637-b8c1e707";
+  const artefactTypesPacketId = "20240729-155513-1432bfa7";
+
+  test("can see packet group name and packet id", async ({ page }) => {
+    await page.goto(`./parameters/${parametersPacketId}`);
+    const content = await getContentLocator(page);
+    await expect(await content.getByRole("heading", { name: "parameters", level: 2 })).toHaveText("parameters");
+    await expect(await content.getByText(parametersPacketId)).toHaveText(parametersPacketId);
+  });
+
+  test.describe("Summary", () => {
+    test("can see parameters", async ({ page }) => {
+      await page.goto(`./parameters/${parametersPacketId}`);
+      const content = await getContentLocator(page);
+      const parametersDiv = await getPacketPageAccordionSection(content, "Parameters");
+      await expect(parametersDiv.getByText("a: 1")).toBeVisible();
+      await expect(parametersDiv.getByText("b: hello")).toBeVisible();
+      await expect(parametersDiv.getByText("c: false")).toBeVisible();
+    });
+
+    test("can see dependencies", async ({ page }) => {
+      await page.goto(`/depends/${dependsPacketId}`);
+      const content = await getContentLocator(page);
+      const dependsDiv = await getPacketPageAccordionSection(content, "Dependencies");
+      const depItems = await dependsDiv.getByRole("listitem");
+      await expect(depItems).toHaveCount(2);
+      const dep1Name = "explicit";
+      const dep1Id = "20240729-154639-25b955eb";
+      await expect(await depItems.first()).toHaveText(`${dep1Name}${dep1Id}`);
+      await expect(await depItems.first().getByRole("link")).toHaveAttribute("href", `/${dep1Name}/${dep1Id}`);
```

Comment:
It would be nice to use `getPackitInstance` (or equivalent) consistently so we can run this against `packit-dev` as well.